_episodes/02-software-development.md

```diff
@@ -19,24 +17,137 @@ View (user interface), and Controller (input/output and data manipulation)."
+entry point in the application - we will have a more detailed look into it shortly. 
+On closer inspection, we can see that the `inflammation` directory contains two more Python scripts - 
+`view.py` and `model.py`.
+
+~~~
+$ ls -l inflammation
+total 24
+-rw-r--r--  1 alex  staff  838 29 Jun 09:59 models.py
+-rw-r--r--  1 alex  staff  649 25 Jun 13:13 views.py
+~~~
+{: .language-bash}
+
+Directory `data` contains several files with patients’ daily inflammation information. 
+
+~~~
+$ ls -la data 
+total 264
+drwxr-xr-x  18 alex  staff    576 25 Jun 13:13 .
+drwxr-xr-x  16 alex  staff    512 30 Jun 14:23 ..
+-rw-r--r--   1 alex  staff   5365 25 Jun 13:13 inflammation-01.csv
+-rw-r--r--   1 alex  staff   5314 25 Jun 13:13 inflammation-02.csv
+-rw-r--r--   1 alex  staff   5127 25 Jun 13:13 inflammation-03.csv
+-rw-r--r--   1 alex  staff   5367 25 Jun 13:13 inflammation-04.csv
+-rw-r--r--   1 alex  staff   5345 25 Jun 13:13 inflammation-05.csv
+-rw-r--r--   1 alex  staff   5330 25 Jun 13:13 inflammation-06.csv
+-rw-r--r--   1 alex  staff   5342 25 Jun 13:13 inflammation-07.csv
+-rw-r--r--   1 alex  staff   5127 25 Jun 13:13 inflammation-08.csv
+-rw-r--r--   1 alex  staff   5327 25 Jun 13:13 inflammation-09.csv
+-rw-r--r--   1 alex  staff   5342 25 Jun 13:13 inflammation-10.csv
+-rw-r--r--   1 alex  staff   5127 25 Jun 13:13 inflammation-11.csv
+-rw-r--r--   1 alex  staff   5340 25 Jun 13:13 inflammation-12.csv
+-rw-r--r--   1 alex  staff  22554 25 Jun 13:13 python-novice-inflammation-data.zip
+-rw-r--r--   1 alex  staff     12 25 Jun 13:13 small-01.csv
+-rw-r--r--   1 alex  staff     15 25 Jun 13:13 small-02.csv
+-rw-r--r--   1 alex  staff     12 25 Jun 13:13 small-03.csv
+~~~
+{: .language-bash}
+
+The data is stored in
+a series of comma-separated values (CSV) format files, where:
+
+- each row holds temperature measurements for a single patient (in some arbitrary units of inflammation),
+- columns represent successive days.
+
+You can inspect the data files from the command line by issuing command `cat`, e.g. do `cat data/inflammation-01.csv` 
+from the project root to check the contents of the first inflammation file.
+
+Directory `tests` contains several tests that have been implemented already. We will be adding more tests 
+during the workshop as our code grows.
+
+An important thing to note here is that the structure of the project is not arbitrary. 
 One of the big differences between novice and intermediate software development is planning the structure of your code.
 A novice will often make up the structure of their code as they go along. However, for more advanced software development,
-we need to plan this structure - called a *software architecture* - beforehand.
-We’ll look into the Model-View-Controller architecture, one we’ll use in the example software
-project we’ll develop over the course of this workshop.
+we need to plan this structure - called a *software architecture* - beforehand. Let's have a more detailed look
```

Comment:
One way this could be misconstrued is that a directory structure, by itself, represents a software architecture (where an architecture is more than just the file structure, it's also about interfaces, behavioural interactions between components, etc.) Perhaps rephrase to be more general, and mention a directory structure is part of that?

Reply:
Addressed via 9961fc6.